### PR TITLE
[Synapse Data Plane]Add vnet feature to data plane

### PR DESCRIPTION
**Pull request by aim-for-better**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/managedPrivateEndpoints.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+          }
+        }
+      }
+    },
+    "/managedVirtualNetworks/{managedVirtualNetworkName}/managedPrivateEndpoints": {
+      "get": {
+        "tags": [
+          "ManagedPrivateEndpoints"
+        ],
+        "operationId": "ManagedPrivateEndpoints_List",
```
Comment (jhendrixMSFT) [inline specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/managedPrivateEndpoints.json:139]: Looks like this returns a pageable response?  If so can you please add [x-ms-pageable](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable)

Comment (aim-for-better) [reply specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/managedPrivateEndpoints.json:139]: Added

---
In reply to: [408458563](https://github.com/Azure/azure-rest-api-specs/pull/8753#discussion_r408458563) [](ancestors = 408458563)

**Comment by jhendrixMSFT** [comment]
@aim-for-better is this functionality live and ready for consumption?

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by aim-for-better** [comment]
Could you please help us merge it into master branch?

**Comment by jhendrixMSFT** [comment]
@aim-for-better I need confirmation that the changes to the service are live before I can merge.

**Comment by aim-for-better** [comment]
> @aim-for-better I need confirmation that the changes to the service are live before I can merge.

Hi Joel, I have confirmed with backend team that the related APIs are live. Thank you very much.

**Pull request merged**